include/flatbuffers/reflection_generated.h

```diff
@@ -161,7 +161,6 @@ struct TypeBuilder {
         : fbb_(_fbb) {
     start_ = fbb_.StartTable();
   }
-  TypeBuilder &operator=(const TypeBuilder &);
```

Comment:
I agree, `flatbuffers::FlatBufferBuilder &fbb_;` blocks copy and move even if there was an idea to make `&operator=` private.